setup.py

```diff
@@ -101,9 +101,9 @@ def get_git_head_sha(base_dir):
 
 def get_build_version(xla_git_sha):
   version = os.getenv('TORCH_XLA_VERSION', '2.2.0')
-  if _check_env_flag('VERSIONED_XLA_BUILD', default='0'):
+  if _check_env_flag('VERSIONED_XLA_BUILD', default='1'):
```

Comment:
Since this variable looks otherwise unused, what do you think of changing the name to be more explicit, e.g. `APPEND_GIT_SHA`?

"`VERSIONED_XLA_BUILD` makes me think of releases, which is the opposite of what we want.

Reply:
Sure, why not.